apps/rush/src/taskRunner/ProjectBuildTask.ts

```diff
@@ -221,7 +217,32 @@ export default class ProjectBuildTask implements ITaskDefinition {
     }
   }
 
-  private _getScriptCommand(script: string): { command: string, args: string[] } {
+  private _getBuildCommand(): IBuildCommand {
+    const build: IBuildCommand =
+      this._getScriptCommand('test') || this._getScriptCommand('build');
+
+    if (!build) {
+      // tslint:disable-next-line:max-line-length
+      throw new Error(`The project [${this._rushProject.packageName}] does not define a 'test' or 'build' command in the 'scripts' section of its package.json`);
+    }
+
+    // Normalize test command step
+    build.args.push(this._errorDisplayMode === ErrorDetectionMode.VisualStudioOnline ? '--no-color' : '--color');
+
+    if (this._production) {
+      build.args.push('--production');
+    }
+    if (this._npmMode) {
+      build.args.push('--npm');
+    }
+    if (this._minimalMode) {
+      build.args.push('--minimal');
+    }
```

Comment:
we have a bug for it, but definitely out of scope for this PR

---
In reply to: [116883375](https://github.com/Microsoft/web-build-tools/pull/195#discussion_r116883375) [](ancestors = 116883375)